selectors/emojis.test.js

```diff
@@ -22,13 +20,11 @@ describe('Selectors.Emojis', () => {
         assert.deepEqual(Selectors.getRecentEmojis(testState), []);
 
         const recentEmojis = ['rage', 'nauseated_face', 'innocent', '+1', 'sob', 'grinning', 'mm'];
-        LocalStorageStore.setRecentEmojis(userId1, recentEmojis);
         setTimeout(() => {
```

Comment:
These tests should be updated so that the state changes between calls to the selector. The `setTimeout`s also aren't needed any more since they were only to allow for us to write to storage